docs/topics/patching-nodes.md

```diff
@@ -0,0 +1,41 @@
+# Patching Cluster Nodes
+
+To favor a consistent behavior, AKS Engine's approach to infrastructure management is immutability.
+By default, cluster nodes are not patched on bootstrap nor on a regular basis. 
+
+To prevent node patching, AKS Engine creates file `/etc/apt/apt.conf.d/99periodic` with the following content:
+
+```
+APT::Periodic::Update-Package-Lists "0";
+APT::Periodic::Download-Upgradeable-Packages "0";
+APT::Periodic::AutocleanInterval "0";
+APT::Periodic::Unattended-Upgrade "0";
+```
+
+## Enable Node Patching
+
+AKS Engine allows changes to the default node patching configuration in situations where applying the latest security patches beats the benefits of immutable infrastructure.
+
+Setting `linuxProfile.enableUnattendedUpgrades` to `"true"` configures each Linux node VM (including control plane node VMs) to run `/usr/bin/unattended-upgrade` in the background according to a daily schedule. If enabled, the default `unattended-upgrades` package configuration will be used as provided by the Ubuntu distro version running on the VM. More information [here](https://help.ubuntu.com/community/AutomaticSecurityUpdates).
```

Comment:
Super nitpicking, can we change the structure of this doc to be: 

## Enable Node Patching

AKS Engine allows changes to the default node patching configuration in situations where applying the latest security patches beats the benefits of immutable infrastructure.

Option 1: Setting `linuxProfile.enableUnattendedUpgrades` to `"true"` configures each...

Option 2: Enable Node Patching on Bootstrap Only. An intermediate solution is to set `linuxProfile.runUnattendedUpgradesOnBootstrap` to `"true"...

## Orchestrate Node Reboots

Node patches will likely require reboots, to ensure that the nodes are rebooted in a non-disruptive way....